common/Strings/en-us/Resources.resw

```diff
@@ -205,33 +205,49 @@
-    <value>Unable to add user to {0} administrators group. Please try manually.</value>
-    <comment>Locked="{0}" Text explaining that the user add operation to {0} administrators group failed.</comment>
+  <data name="RestartAfterChangesMessage" xml:space="preserve">
+    <value>Changes completed successfully. Please restart your machine for the changes to take effect</value>
+    <comment>Text explaining that a restart is needed after enabling the Hyper-V feature or adding the user to the Hyper-V Admin group</comment>
   </data>
-  <data name="UserNotInHyperAdminGroupButton" xml:space="preserve">
-    <value>Launch computer management</value>
-    <comment>Button text for the user to click, in order to launch computer management</comment>
+  <data name="HyperVAdminAddUserAndEnableHyperVMessage" xml:space="preserve">
+    <value>The current user is not in the Hyper-V administrators group and the Hyper-V feature is not enabled. Hyper-V virtual machines will not load. To resolve both issues please click the set up Hyper-V in Dev Home button and reboot</value>
+    <comment>Locked="{Hyper-V}" Text explaining that the user is not in the Hyper-V administrators group and the Hyper-V feature is not enabled</comment>
   </data>
   <data name="UserNotInHyperAdminGroupMessage" xml:space="preserve">
-    <value>The current user is not a {0} administrators. {0} virtual machines will not load. Please add the user to the {0} administrators group and reboot.</value>
-    <comment>Locked="{0}" Text explaining that the user is not in the {0} administrators group and that we need to add them.</comment>
+    <value>The current user is not in the Hyper-V administrators group. Hyper-V virtual machines will not load. To resolve the issue, click the add user to group button</value>
+    <comment>Locked="{Hyper-V}" Text explaining that the user is not in the Hyper-V administrators group</comment>
+  </data>
+  <data name="HyperVFeatureNotEnabledMessage" xml:space="preserve">
+    <value>The Hyper-V feature is not enabled. Hyper-V virtual machines will not load. To resolve the issue, click the enable Hyper-V button</value>
+    <comment>Locked="{Hyper-V}" Text explaining that the Hyper-V feature is not enabled</comment>
+  </data>
+  <data name="UnableToEnableHyperVFeatureMessage" xml:space="preserve">
+    <value>Unable to enable the Hyper-V feature. Hyper-V virtual machines will not load. To resolve the issue manually, enable Hyper-V manually and reboot</value>
+    <comment>Locked="{Hyper-V}" Text explaining that we were not able to enable the Hyper-V feature.</comment>
+  </data>
+  <data name="UnableToAddUserToHyperVAdminAndEnableHyperVMessage" xml:space="preserve">
+    <value>Unable to add the user to the Hyper-V administrators group or enable Hyper-V. Hyper-V virtual machines may not load. To resolve the issue manually, add the user to the Hyper-V administrators group, enable Hyper-V and reboot</value>

```

Comment:
Can a user be added to the Hyper-V Admin group before enabling Hyper-V?

Reply:
yea they can. The group exists even if Hyper-V isn't enabled